wazuh/wazuh-dashboard-plugins
# Fix statistics visualizations when cluster mode is disabled

### Description
This PR solves the followins issue in `Server Management -> Statistics`:
- Fix initial error when cluster mode is disabled.
- Replace the definitions of visualizations `Events processed` and `Events Dropped` according to the cluster mode, and this way only add the aggregation by the field `node.name` when cluster mode is enabled.
- Apply visualizations filter when node is selected.

 
### Issues Resolved
https://github.com/wazuh/wazuh-dashboard-plugins/issues/6343

### Evidence
#### Cluster mode disabled
![Cluster mode disabled](https://github.com/wazuh/wazuh-dashboard-plugins/assets/103193307/289c1329-2acb-4988-86a0-21db147a4b8f)

#### Cluster mode enabled
![Cluster mode enabled](https://github.com/wazuh/wazuh-dashboard-plugins/assets/103193307/9edc1a78-81b5-4bdc-b827-2f7c8e02fa6c)

#### Cluster mode enabled and filtered by node
![Filtered by node](https://github.com/wazuh/wazuh-dashboard-plugins/assets/103193307/476f4786-a7e0-4928-8b44-ec29724fda05)

### Test

1. Disabled cluster mode from the server configuration: https://documentation.wazuh.com/current/user-manual/configuring-cluster/index.html
2. Go to `Server Management -> Statistics`.
3. Check the visualizations from both tabs.
4. Enabling cluster mode and repeat steps 2 and 3.
5. Filter by node and check visualizations.

### Check List
